aspnet/migration/rc1-to-rtm.rst

```diff
@@ -0,0 +1,498 @@
+Migrating from ASP.NET 5 RC1 to ASP.NET Core 1.0
+================================================
+
+By `Cesar Blum Silveira`_, `Rachel Appel`_, `Rick Anderson`_ 
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+
+ASP.NET 5 RC1 apps were based on the .NET Execution Environment (DNX) and made use of DNX specific features. ASP.NET Core 1.0 is based on .NET Core, so you must first migrate your application to the new .NET Core project model. See `migrating from DNX to .NET Core CLI <http://dotnet.github.io/docs/core-concepts/dnx-migration.html>`_ for more information.
+
+See the following resources for breaking changes, announcements and migrations information:
+
+- `ASP.NET Core RC2 breaking changes <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0-rc2>`_ 
+- `ASP.NET Core 1.0 breaking changes <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0>`_
+- `Upgrading from Entity Framework RC1 to RC2 <https://docs.efproject.net/en/latest/miscellaneous/rc1-rc2-upgrade.html>`_.
+- :doc:`/migration/rc2-to-rtm`
+
+Update Target Framework Monikers (TFMs)
+---------------------------------------
+
+If your app targeted ``dnx451`` or  ``net451`` in the ``frameworks`` section of *project.json*, you must make the following changes:
+
+==================================== ====================================
+DNX                                  .NET Core
+==================================== ====================================
+``dnx451``                           ``net451``
+``dnxcore50``                        ``netcoreapp1.0``
+==================================== ====================================
+
+.NET Core apps must add a dependency to the ``Microsoft.NETCore.App`` package:
+
+.. code-block:: json
+  :emphasize-lines: 4-8
+
+  {
+    "frameworks": {
+      "netcoreapp1.0": {
+        "dependencies": {
+          "Microsoft.NETCore.App": {
+            "version": "1.0.0",
+            "type": "platform"
+          }
+        }
```

Comment:
@Eilon @victorhurdugaci @JunTaoLuo  That's not what my 14.1.20624.0 build (latest) generates. This is what I get
```
"frameworks": {
    "netcoreapp1.0": {
      "imports": [
        "dotnet5.6",
        "portable-net45+win8"
      ]
    }
  },
```